data/README.md

```diff
@@ -0,0 +1,28 @@
+# The `data` Directory
+
+## Overview
+
+This directory contains files used for pre-loading our production server with
+*Pseudomonas aeruginosa*-related samples and annotations. All of these files
+are loaded into the system using the `import_data_and_index()` function in
+[fabfile/adage_server.py](https://github.com/greenelab/adage-server/blob/master/fabfile/adage_server.py).
+The
+[adage/adage/configy.py.template file](https://github.com/greenelab/adage-server/blob/master/adage/adage/config.py.template)
+specifies which files are loaded by the `import_data_and_index()` function.
+
+## File Listing
+
+* **PseudomonasAnnotation.tsv** lists samples retrieved from ArrayExpress along
+  with manually-curated annotations for those samples.
+* **all-pseudomonas-gene-normalized.pcl** contains normalized expression levels for
+  every gene in each sample in the compendium.
```

Comment:
Can you put a little more information here? For example, it might be helpful to specify that genes are rows, samples are columns. I would also consider stating that this is the input expression data for building an ADAGE model.